### ISSUE TITLE
Race condition in RedisModel.get

### ISSUE THREAD

**Issue by twidi** [limpyd/redis-limpyd#82]
Considere these lines of the `get` method of the `RedisModel`:

```python
result = cls.collection(**kwargs).sort(by='nosort')
if len(result) == 0:
    raise DoesNotExist(u"No object matching filter: %s" % kwargs)
elif len(result) > 1:
    raise ValueError(u"More than one object matching filter: %s" % kwargs)
else:
    pk = result[0]
```

The first line will do nothing in redis, as a collection is lazy
The second line will do the needed intersections and do a `scard` on the result to get the count
Assume the object exists in limpyd, so `len` will return `1`, and so we jump to the last line.
This last line do the needed intersections, again, and return the first entry of the result list.

Ok.

But, and it's not impossible (it happened to me twice tonight), if another process delete, or update the object so that it doesn't macht the filter anymore, the last line will raise a `IndexError: list index out of range`...

We have a race condition.

**Comment by twidi** [comment]
Fixed on develop

**Comment by twidi** [comment]
Fixed in 1.1
